### ISSUE TITLE
Add HelloWorld Programs in different languages

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/HelloWorld#1]
This issue is specfically created for all the folks out there who wants to contribute to open source.

Add HelloWorld program in different languages.

Repository : Create a folder of the language you are programming
Level : Easy

Do check if the code you want to contribute is available in this repository
Comment on this issue with the language you are going to contribute
Start working on the script and before you add make sure it works properly
Add your code to your created repository using the contribution guidelines mentioned
Create a Pull Request and be patient
I'll review your code and merge your PR as early as possible.

Any queries, you can drop a message here. I'll be happy to respond.
Best of luck everyone ❤️

Happy Contribution!

**Comment by rarahim75** [comment]
Hi, as a tribute to my first programming language i learnt back in 1997, i plan to contribute a Pascal version of the Hello World program. Is that ok?

**Comment by prathimacode-hub** [comment]
Yes, absolutely ok. I have created a repository of Pascal. You can add your code. And file name should look like 👍 
 
HelloWorld_FirstName.yourprogramextension  @rarahim75

**Comment by rarahim75** [comment]
> Yes, absolutely ok. I have created a repository of Pascal. You can add your code. And file name should look like 👍
> 
> HelloWorld_FirstName.yourprogramextension @rarahim75

Ok noted, tq.

**Comment by Amisha328** [comment]
Hey, @prathimacode-hub 
I am familiar with C++, Java, and Python. 
Can I contribute in all three mentioned languages?

**Comment by prathimacode-hub** [comment]
Yes, you can. HelloWorld_YourFirstName.extension and that's it. You can contribute in how much ever languages you want to. As you include your name in your file. This is the place for you to initiate and learn contributing. 

@Amisha328

**Comment by Amisha328** [comment]
Okay, thank you @prathimacode-hub

**Comment by prathimacode-hub** [comment]
> > Yes, absolutely ok. I have created a repository of Pascal. You can add your code. And file name should look like 👍
> > HelloWorld_FirstName.yourprogramextension @rarahim75
> 
> Sorry for asking obvious questions but this is after all going to be my first _hopefully_ successful pull request so I'm still not very familiar with github workflows. What did you mean you created a repository of Pascal? where can I access it? shouldn't I be following the contributing guidelines outlined [here](https://github.com/prathimacode-hub/HelloWorld/blob/main/CONTRIBUTING.md)?

Sorry, long back. Just checked on with your message. Yes, those are the contributing guidelines. I had create a repository of Pascal, so that you could add the HelloWorld program in Pascal. You can see the "Pascal" folder in this project. Inside it, your file name should be as "HelloWorld_YourFirstName.extension"

If you need any help with guidance on github workflow, you can reach out to me here. Don't hesitate. @rarahim75

**Comment by pankaj892** [comment]
@prathimacode-hub I would like to add Hello World in Scala and Go Language. Please assign me

**Comment by prathimacode-hub** [comment]
Sure. But it won't come under hacktober fest ok? @pankaj892

**Comment by pankaj892** [comment]
Yeah @prathimacode-hub that's not a problem I am here to contribute not to get a tshirt 😁

**Comment by dipayan22** [comment]
Hii @prathimacode-hub , I want to contribute rust programing language here. Cab you please give me this issue .
